LeaderboardBackend.Test/Controllers/UsersControllerTests.cs

```diff
@@ -50,8 +50,7 @@ public async Task GetUser_NotFound_UserDoesNotExist()
 			.Setup(x => x.GetUser(It.IsAny<Guid>()))
 			.Returns(Task.FromResult<User?>(null));
 
-		ActionResult<User> response = await _controller.GetUser(defaultUserId);
-
+		var response = await _controller.GetUser(defaultUserId);
```

Comment:
Only asking this cuz I'm still new to C#; may I know why you've decided to change to `var` here?

Reply:
The call on `_controller` is `GetUser`. It's obvious here that the *response* will be a `User` (specifically an `ActionResult<User>`, but that's hardly important, since the response is used exactly one time, where the taken parameter must already be an `ActionResult`).

Reply:
I see. So since we know what we're getting, and we don't reuse the variable a lot, you've decided to set em as `var`s then.